pkgs/tools/typesetting/lowdown/default.nix

```
@@ -14,6 +14,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ which ]
     ++ lib.optionals stdenv.isDarwin [ fixDarwinDylibNames ];
 
+  preConfigure = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64)
```

Please use this style:
```suggestion
  preConfigure = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64) ''
    echo 'HAVE_SANDBOX_INIT=0' > configure.local
  ''
```
There is a subtle edge case with overrides where adding `""` strings together (imagine adding extra `preConfigure` steps in an override) will result a missing newline and broken bash code.